publicodes/source/units.ts

```diff
@@ -254,7 +254,7 @@ function simplifyUnitWithValue(unit: Unit, value = 1): [Unit, number] {
 	]
 }
 export function areUnitConvertible(a: Unit | undefined, b: Unit | undefined) {
-	if (a == null || b == null) {
+	if (a == null && b == null) {
```

Comment:
C'est une grosse modification. Sur le principe je suis pas opposé, mais ça implique pas mal de choses, et il y a peut-être d'autres endroit à modifier.

Reply:
Complétement d'accord ! J'ai fait cette modification sans considération suffisante car 62320d1 ne détectait pas le manque d'unité sur `salaire médian: 2300` (qui pour rappel provoque un bug sur le simulateur annuel) sans ça et que les tests passaient. Je vais annuler ce changement pour le moment et ouvrir une issue dédiée pour la vérification stricte de l'homogénéité des formules.